helpers/mathjax.js

```diff
@@ -62,24 +61,6 @@ class HtmlBlockMathRenderer {
 			elm.style.height = '0.5rem';
 		});
 
-		if (context.enableMML3Support) {
-			// There's a bug in the experimental MML3 plugin that causes mi and mo elements containing non-breaking
-			// spaces to break MathJax's math processing (e.g. <mo>&nbsp;</mo>, <mi>Some&nbsp;Identifier</mi>).
-			// Unfortunately, WIRIS tends to add a lot of these in chemistry equations, so they break math processing.
-			//
-			// In order to address this, we can just remove any non-breaking spaces entirely, replacing them with
-			// empty strings. MathJax will ignore any empty elements as a result, and while this may mean intended
-			// whitespace is occasionally removed, it's necessary for MathJax to render anything at all.
-			//
-			// NOTE: MathJax evidently has a fix for this in MathJax 4, so we should consider trying to remove this when
-			// the update comes out of beta and we decide to take it on.
-			//
-			// See https://github.com/mathjax/MathJax/issues/3030 for some related discussion.
```

Comment:
Context for other reviewers: It turned out there were a number of other bugs in this implementation and attempts to find workarounds were not particularly successful (they don't play nicely with our shadowDOM adapter stuff). So we're just removing this feature entirely instead.